pages/formester-vs-microsoft-forms.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <comparision-hero
       heading="Switch to <span class='hglight'>Formester</span> and craft interactive web forms – Perfect Alternative to Microsoft Forms"
-      description="Ditch the dry design and limited integrations of Microsoft Forms. Optimise your data collection with beautiful, personalised forms that captivate audiences, streamline orders, and automate workflows like magic. <span class='hglight'>Forget coding headaches - our intuitive drag-and-drop interface empowers everyone to be a form-building pro.</span>"
+      description="Ditch the dry design and limited integrations of Microsoft Forms. Optimise your data collection with beautiful, personalised forms that captivate audiences, streamline orders, and automate workflows like magic. Forget coding headaches - our intuitive drag-and-drop interface empowers everyone to be a form-building pro."
```

Comment:
Is highlight not required in the new description?

Reply:
yes Siddharth told to remove this